fasthtml/core.py

```diff
@@ -439,7 +440,7 @@ def __str__(self): return p
 class FastHTML(Starlette):
     def __init__(self, debug=False, routes=None, middleware=None, exception_handlers=None,
                  on_startup=None, on_shutdown=None, lifespan=None, hdrs=None, ftrs=None,
-                 before=None, after=None, ws_hdr=False,
+                 before=None, after=None, ws_hdr=False, ct_hdr=False,
```

Comment:
What does `ct_hdr` stand for? Chunked Transfer Header?

Reply:
exactly!

Reply:
should we rename it? i wasn't able to come up with something clearer

Reply:
I like the name. Just wanted to be explicit for when I document it. 😄